ipv6/miredo/Makefile

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (C) 2006-2017 OpenWrt.org
```

Comment:
OK, I probably misunderstood

> An up-to-date copyright notice. Use OpenWrt if no other present or supply your own.

in the Contribution Guidelines. I don't really care for copyright, so I'm happy to have none, but I can put there whatever is preferred, or create my own.